plugins/modules/grafana_organization_user.py

```diff
@@ -0,0 +1,199 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Copyright: (c) 2021
+
+from __future__ import absolute_import, division, print_function
+
+DOCUMENTATION = '''
+---
+module: grafana_organization_user
+author:
+  - Aliaksandr Mianzhynski (@amenzhinsky)
+version_added: "1.5.0"
+short_description: Manage Grafana Organization Users.
+description:
+  - Add or remove users or change their roles in Grafana organizations through org API.
```

Comment:
Can you specify if the user needs to exist before using this module ? and add a link to the grafana_user if they need to be created before using it ?